### PR TITLE
Absolute pathing for brigdoors.dm

### PR DESCRIPTION
**Pull request by ShiftyRail**
This also removes trailing returns and a duplicate `update_icon` definition, as well as a logic function being not used.

### CODE REVIEW

File: code/game/machinery/doors/brigdoors.dm [hunk 1]
```diff
@@ -23,147 +23,137 @@
...
+		if(F.id_tag == src.id_tag)
+			targets += F
 
-		for(var/obj/structure/closet/secure_closet/brig/C in brig_lockers)
-			if(C.id_tag == src.id_tag)
-				targets += C
-
-		if(targets.len==0)
-			stat |= BROKEN
-		update_icon()
```
Comment (ShiftyRail) [inline code/game/machinery/doors/brigdoors.dm:46]: Wait, this was not a duplicate def, but indeed a function call, let me fix that

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -688,6 +688,7 @@ About the new airlock wires panel:
 /obj/machinery/door/airlock/Topic(href, href_list, var/nowindow = 0)
 	// If you add an if(..()) check you must first remove the var/nowindow parameter.
 	// Otherwise it will runtime with this kind of error: null.Topic()
+
```
Comment (DamianX) [inline code/game/machinery/doors/airlock.dm]: nice

Comment (ShiftyRail) [reply code/game/machinery/doors/airlock.dm]: I like my white spaces.

File: code/game/machinery/doors/brigdoors.dm [hunk 3]
```diff
@@ -23,147 +23,138 @@
...
+		timer_end()
 
 
 // has the door power sitatuation changed, if so update icon.
-	power_change()
-		..()
-		update_icon()
-		return
+/obj/machinery/door_timer/power_change()
+	..()
```
Comment (ShiftyRail) [inline code/game/machinery/doors/brigdoors.dm]: I slashed the update_icon here

Comment (DamianX) [reply code/game/machinery/doors/brigdoors.dm]: why

Comment (ShiftyRail) [reply code/game/machinery/doors/brigdoors.dm]: Mistake
Probably removed it while trying to remove the return
It will be fixed tommorow

**Comment by ShiftyRail** [comment]
Fixed.

**Pull request merged**